Cargo.toml

```diff
@@ -49,11 +49,6 @@ name = "fs_readonly"
 path = "tests/fs_readonly.rs"
 harness = false
 
-[[test]]
-name = "fs_restrict_child"
-path = "tests/fs_restrict_child.rs"
```

Comment:
This test was removed, but the file has not been deleted.

Reply:
Feels like it might have been a rebase mishap.